index.html

```diff
@@ -0,0 +1,171 @@
+                       корм на Cat Energy Slim. Отличный результат без изнуряющих тренировок! При этом он не 
+                       менял своих привычек и по-прежнему спит по 16 часов в день.</p>
+                       <div class="example__means">
+                           <div class="example__mean">
+                               <p class="example__main-text">5 кг</p>
+                               <p class="example__secondary-text">снижение веса</p>
+                           </div>
+                           <div class="example__mean">
+                            <p class="example__main-text">60 дней</p>
+                            <p class="example__secondary-text">затрачено времени</p>
+                            </div>
+                       </div>
+                       <div class="example__total">
+                           <p class="example__expenses">Затраты на питание: </p>
+                           <p class="example__sum">15 000 руб.</p>
+                       </div>
+              </div>
+              <div class="example__before-after">
+                  <img
+                  class="example__weight-cat"
+                  src="./img/before-desktop.svg"
+                  width="365"
+                  height="536"
+                  alt="before_cat"
+                />
+                <img
+                class="example__weight-cat"
+                src="./img/after-desktop.svg"
+                width="325"
+                height="654"
+                alt="after_cat"
+              />
+              </div>
+          </div>
+      </section>
+
+      <section class="map">
+        <div class="map__content">
+          <p class="map__cooperation">Приглашаем 
+            к сотрудничеству 
+            дилеров!</p>
+            <div class="map__text-wrap">
+              <p class="map__adress">
+                ул. Большая Конюшенная, д. 19/8
+              </p>
+              <p class="map__country">
+                Санкт-Петербург
+              </p>
+            </div>
+        </div>
+      </section>
+    </main>
+    <footer class="footer">
+      <div class="footer-wrapper container">
+        <img class="footer__logo" src="./img/footer-logo.svg" width="128" height="24" alt="footer-logo">
+        <ul class="footer-socials">
+          <li class="footer__list"><a class="footer__link" href="#"><img class="footer__vk" src="./img/vk.svg" width="30" height="16" alt="vk"></a></li>
+          <li class="footer__list"><a class="footer__link" href="#"><img class="footer__inst" src="./img/insta.svg" width="23" height="23" alt="instagram"></a></li>
+          <li class="footer__list"><a class="footer__link" href="#"><img class="footer__fb" src="./img/fb.svg" width="14" height="28" alt="facebook"></a></li>
+        </ul>
+        <span class="footer__academy"><p class="footer__html-academy">HTML Academy</p>
```

Comment:
1. As far as I remember, we can't nest a `p` inside a `span`. A paragraph should be a larger structure. We'd better use a `div` here.
2. Calling a CSS class `academy` or `html-academy` is not very proper. It's rather a `copyright`, `copyright__text`, `copyright__link`, `copyright__logo` — it's okay to create a new BEM block for it, like this:
`footer__copyright copyright`

Reply:
++